src/js.cookie.js

```diff
@@ -146,16 +135,16 @@
 		api.getJSON = function () {
 			return api.apply({
 				json: true
-			}, [].slice.call(arguments));
+			}, arguments);
```

Comment:
Could this be problematic in older browsers?

Reply:
We don't run the tests on IE 6 and 7 anybody [because of reasons](https://github.com/js-cookie/js-cookie/issues/370#issuecomment-340137482). IE 8 seems to be working, though.

Need to manually test in those browsers I guess because the version we're working on still supports it in the docs.

Reply:
Fwiw, the only reason I could think of why we had the arguments to array trick in there was because of JavaScript engines in these old browsers. Unfortunately I can’t remember whether IE had a problem with that or not.